src/Nethermind/Nethermind.Test.Runner/StateTestTxTracer.cs

```diff
@@ -110,7 +109,28 @@ public void SetOperationStack(TraceStack stack)
     public void SetOperationMemory(TraceMemory memoryTrace)
     {
         if (IsTracingDetailedMemory)
-            _traceEntry.Memory = string.Concat("0x", string.Join("", memoryTrace.ToHexWordList().Select(mt => mt.Replace("0x", string.Empty))));
+        {
+            var length = 2;
+            var wordList = memoryTrace.ToHexWordList();
```

Comment:
potentially you could move this under memory trace instead of `ToHexWordList` and not create that list in the first place, would be even faster